effekt/jvm/src/test/scala/effekt/LLVMTests.scala

```diff
@@ -12,105 +12,102 @@ class LLVMTests extends EffektTests {
 
   override lazy val positives: List[File] = List(
     examplesDir / "llvm",
+    examplesDir / "pos",
+  )
+
+  lazy val bugs: List[File] = List(
+    examplesDir / "pos" / "issue108.effekt", // seg faults!
+
+    // boxing
+    examplesDir / "benchmarks" / "church_exponentiation.effekt", // ... are used as type parameters but would require boxing.
+    examplesDir / "llvm" / "not-boxed.effekt", // Unboxing is performed on not-boxed int, leading to program termination
```

Comment:
@marzipankaiser could you help me looking into these bugs, which are related to polymorphism boxing not treating first-class functions correctly at the moment?

Reply:
Actually, the second one is the blocker for concurrent IO with libuv, which I minimized